apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -312,16 +319,19 @@ class VotePanelContent extends React.Component {
                       No
                     </VotingButton>
                   </ButtonsContainer>
-                  <Info.Action>
-                    <p>
-                      You will cast your vote with{' '}
-                      {userBalance === null
-                        ? '… tokens'
-                        : pluralize(userBalance, '$ token', '$ tokens')}
-                      , since it was your balance at the beginning of the vote{' '}
-                      {this.renderBlockLink(snapshotBlock)}.
-                    </p>
-                  </Info.Action>
+                  {/* This check is until we get the connect button in the MenuPanel */
+                  userBalance !== null && (
```

Comment:
Maybe we could detect that there's no account and instead say "You will need to connect your account to vote"?

Reply:
Yes good idea! Also, what would you think about a way for apps to request the account connection?

Reply:
We could certainly bake that into aragon.js 👍 : https://github.com/aragon/aragon.js/issues/237